sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -13,8 +16,24 @@ internal static class Statsbeat
 
         private const string StatsBeat_ConnectionString = "<StatsBeat_ConnectionString>";
 
+        private const string AMS_Uri = "http://169.254.169.254/metadata/instance/compute";
```

Comment:
nit: May not need to split the URL, it does not provide any benefit.

Reply:
removed